main.go

```diff
@@ -68,13 +68,11 @@ func main() {
 	}
 
 	if flags.dry == false {
-		resolve := output.Stdout("Attempting to push new tag to GitHub...")
+		output.Stdout("Attempting to push new tag to GitHub...")
```

Comment:
Haha I forget how my code works all the time.

Whatever final choice you want to make I'm fine with, you are the owner of this repo after all :).